libs/blocks/recommended-articles/recommended-articles.js

```diff
@@ -20,10 +20,13 @@ async function getArticleDetails(article) {
   const ending = trimEndings.find((el) => title.endsWith(el));
   [title] = title.split(ending);
 
+  const metaCategory = getMetadata('category', doc);
```

Comment:
If `category` is a new metadata field, I'd make it more specific to this block, to avoid naming clashes with other blocks

Reply:
I definitely agree, but this is kind of an interesting issue. So this metadata property does not exist in `brand-blog`'s content. They use the first meta article tag on each page to set labels. The `category` metadata property exists in `dx-blog` on every article currently authored and was used for their labels, meaning it is already a locked in convention. 

I am not so sure recommended articles is or will be used elsewhere. Would also like @rgclayton to weigh in since he knows more about `brand-blog`

Reply:
@JasonHowellSlavin
Correct, blog.adobe.com uses the first tag authored/on the page. 

I understand @mokimo's point.  While I don't know of recommended articles being used elsewhere, I think it might be safe to give `category` a bit more meaning to avoid possible naming clash. What about something like `article-category` to help identify what's going on?

Reply:
After speaking to Ryan, we have the go ahead for merging this. Essentially, every article that business-blog has already uses the `category` metadata property, and we cannot at this juncture bulk update all of those pages. 

However, I have created a ticket to address the issue, and discuss switching to a different property name and paradigm to address Okan's and Ryan's points.